### PR TITLE
Fix persistence, move it cosmetically, actually redirect you to / if you have bad auth. 

### PR DESCRIPTION
**Pull request by macrael**

### CODE REVIEW

File: client/src/profile/profileActions.js [hunk 1]
```diff
@@ -28,11 +28,13 @@ export function updateProfile(authToken, newProfile) {
     dispatch(dismissAlert());
   })
   .catch((error) => {
-    dispatch(displayAlert('usa-alert-error', 'Error Saving Profile', 'We were unable to save your profile. Please refresh the page and try again.'));
-    console.error('updateProfile Error: ', error);
-    if (error.isAuthRelatedError) {
-      console.log('we failed the auth check, make them login again.');
-      dispatch(push('/'));
+    if (error.response.status === 403) {
+      console.log('bad user token');
```
Comment (tenshiemi) [inline client/src/profile/profileActions.js]: console.error()

File: client/src/profile/profileActions.js [hunk 2]
```diff
@@ -57,11 +59,15 @@ export function getProfile(authToken) {
...
   .catch((error) => {
-    dispatch(displayAlert('usa-alert-error', 'Error Loading Profile', 'We were unable to load your profile. Please refresh the page and try again.'));
-    console.error('getProfile Error: ', error);
-    if (error.isAuthRelatedError) {
-      console.log('we failed the auth check, make them login again.');
-      dispatch(push('/'));
+    console.log(error.response.status);
+    console.log(error.response);
+    if (error.response.status === 403) {
+      console.log('bad user token');
```
Comment (tenshiemi) [inline client/src/profile/profileActions.js]: console.error()

**Review by tenshiemi — APPROVED** [review]
Update the logs and it's ready to merge!

File: client/src/profile/profileActions.js (hunk 1 above)
Comment (macrael) [reply client/src/profile/profileActions.js]: jinx!

**Pull request merged**